tasks/db_deploy/test/manual_tests/sql/orca_schema_v4/tables/granules.sql

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
This file is not needed since db_deploy will already perform this SQL as part of the migration and creation tests.